apps/Minecraft Java Server/install

```diff
@@ -0,0 +1,189 @@
+                wget https://github.com/adoptium/temurin16-binaries/releases/download/jdk-16.0.2%2B7/OpenJDK16U-jdk_arm_linux_hotspot_16.0.2_7.tar.gz && tar -xzf OpenJDK16U-jdk_arm_linux_hotspot_16.0.2_7.tar.gz
+                mv jdk-16.0.2+7 java-16-temurin-armhf
+                rm -rf OpenJDK16U-jdk_arm_linux_hotspot_16.0.2_7.tar.gz
+                # check if java is working and remove if broken
+                ./java-16-temurin-armhf/bin/java -version || ( warning "The downloaded java 16 version does not work, removing it now..." && rm -rf java-16-temurin-armhf && error "Install failed without Java 16" ) || exit 1
+                java_16_location="java-16-temurin-armhf/bin/java"
+                ;;
+            "Java 17")
+                # download java 17 (unavailable in some debian versions)
+                rm -rf jdk-17.0.1+12
+                rm -rf java-17-temurin-armhf
+                rm -rf OpenJDK17U-jdk_arm_linux_hotspot_17.0.1_12.tar.gz
+                wget https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.1%2B12/OpenJDK17U-jdk_arm_linux_hotspot_17.0.1_12.tar.gz && tar -xzf OpenJDK17U-jdk_arm_linux_hotspot_17.0.1_12.tar.gz
+                mv jdk-17.0.1+12 java-17-temurin-armhf
+                rm -rf OpenJDK17U-jdk_arm_linux_hotspot_17.0.1_12.tar.gz
+                # check if java is working and remove if broken
+                ./java-17-temurin-armhf/bin/java -version || ( warning "The downloaded java 17 version does not work, removing it now..." && rm -rf java-17-temurin-armhf && error "Install failed without Java 17" ) || exit 1
+                java_17_location="java-17-temurin-armhf/bin/java"
+                ;;
+        esac
+        ;;
+    *)
+        error "arch variable is not set, can not continue"
+        ;;
+esac
+
+case "$java_selection" in
+    "Java 8")
+        java_location="$HOME/Minecraft-Java-Server/$java_8_location"
+    ;;
+    "Java 16")
+        java_location="$HOME/Minecraft-Java-Server/$java_16_location"
+    ;;
+    "Java 17")
+        java_location="$HOME/Minecraft-Java-Server/$java_17_location"
+    ;;
+esac
+
+sh -c "cat > $HOME/Minecraft-Java-Server/start-server.sh << _EOF_
+$java_location -jar $HOME/Minecraft-Java-Server/server.jar nogui
+_EOF_"
+chmod +x "$HOME/Minecraft-Java-Server/start-server.sh"
+
+# sign eula without starting server
+echo "eula=true" > eula.txt
+
+mkdir -p "$HOME/.local/share/applications"
+tee "$HOME/.local/share/applications/Minecraft-Java-Server.desktop" <<EOF
+[Desktop Entry]
+Name=Minecraft Java Server
+Exec=$HOME/Minecraft-Java-Server/start-server.sh
+Path=$HOME/Minecraft-Java-Server
+Icon=${DIRECTORY}/apps/Minecraft Java Server/icon-64.png
+Type=Application
+Categories=Game
+Terminal=true
+EOF
+
+
+status "Your IP address is one of the following: 
+$(hostname -I | grep -Eo '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')"
+status "The IP address is used to play on your local network"
```

Comment:
```suggestion
status "The IP address is used to play on your local network."
```

I'm just that kind of person who likes grammar.. :+1:

Reply:
added